src/internal/_flatCat.js

```diff
@@ -29,4 +29,4 @@ module.exports = (function() {
       }
     };
   };
-}());
+})();
```

Comment:
I'm curious as to the rationale for this.  Is this simply the eslint house style?  Is it configurable?

I admit to a minor preference for the other style, `(function() {/* ... */}())` over `(function() {/* ... */})()`, but it's a fairly minor things.  Still, I don't want to change this for no good reason.  And I think "that's what the linter prefers" is no good reason.  If someone can make a compelling argument, or if eslint doesn't even allow my preferred syntax, then I could see just going with the flow.